src/battle_script_commands.c

```diff
@@ -4673,7 +4673,6 @@ static void Cmd_playanimation(void)
         return;
     }
     #endif
```

Comment:
```suggestion
    #endif

```
To bring back the accidently deleted newline.